libs/SalesforceSDKCommon/SalesforceSDKCommon.xcodeproj/xcshareddata/xcschemes/SalesforceSDKCommon.xcscheme

```diff
@@ -37,10 +37,10 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
+      buildConfiguration = "Debug"
```

Comment:
Just taking Xcode's auto-update to this file.  Nothing really added, just moved around.